### ISSUE TITLE
Enterprise Registration - Scroll on first step

### ISSUE THREAD

**Issue by lin-d-hop** [openfoodfoundation/openfoodnetwork#1766]
Stage one of the enterprise registration is not intuitive.

On my 13" screen you cannot see the Continue button at all. It is totally hidden making it unclear to the user what the next step is after they tick the box.

![register](https://user-images.githubusercontent.com/8669848/29240988-68f04702-7f68-11e7-8932-035a62fae89c.png)

Can this be changed so that there is NO scroll on this screen, such that the full red button is displayed to the user. Perhaps something like this....

![registerux](https://user-images.githubusercontent.com/8669848/29241014-fc5fce0e-7f68-11e7-8ceb-ec3306862193.png)

**Comment by sauloperez** [comment]
I experience the same problem. Is there any reason why we placed the content in a fixed-height box in the first place? I'd remove the limitation of the light grey box and allow the content to fill the page as usual.

**Comment by daniellemoorhead** [comment]
I think it's fair to say that there wouldn't be a particularly good reason, so let's change it.

Could we but the angular for newbies label on this?

**Comment by sauloperez** [comment]
I'm not sure there's even angular there. It seems to be a CSS only thing.

**Comment by RachL** [comment]
Hi all ! I've noticed that on my tablet and my mobile the scroll inside this popin is not possible. When I switch the device position, my screen is not high enough to see the popin entirely and therefore I cannot access the button.
In my opinion, the popin should open as a new page / tab in the mobile/tablet browser rather than a popin. What do you think ? Shall I create another issue or can it be part of this one ? Thanks :)

![screenshot_20171119-162814](https://user-images.githubusercontent.com/6525576/32992197-26b368b4-cd48-11e7-9e1d-dbbbfa023cc1.png)

**Comment by sauloperez** [comment]
Ok, I got tired of this issue :sweat_smile: I'm going to find the culprit and fix it.

**Comment by sauloperez** [comment]
I haven't got the chance to look at it yet, so anyone can take it.

**Comment by kristinalim** [comment]
I was able to reproduce the same issue which @RachL reported for small screens. The area is scrollable, but content is hidden beyond the bottom of the visible area:

![20180819145101-enterprise-registration-small-button](https://user-images.githubusercontent.com/2243/44306339-1dde7780-a3bf-11e8-81b7-7cc6660b0491.png)

Work is already in #2544, but the status of this issue needs to be checked with those changes.

**Comment by daniellemoorhead** [comment]
good pick up @kristinalim, I added this issue number to your PR #2544 explanation so we remember to close this again once the PR is merged. 😄

**Comment by kristinalim** [comment]
After #2544 is merged, the button in small screens will already be clickable, but the user would still have to scroll to see this.

Scrolling seems necessary if we keep the contents as they are. Here's the full length of the modal on an example small screen:

![20180910194657-enterprise-registration-small](https://user-images.githubusercontent.com/2243/45295858-b08fb380-b532-11e8-8441-80964cc8e3bf.png)

These would bring the button higher in the page:

1. Have two buttons instead of one for small screens: one above the "You'll need" text and retaining the existing one below.
2. Collapse the text under headings "What are the requirements?" and "What do I get?"

The first one would be easier, but the second one would give a better experience I think.

I'm not a UX expert. :grin: Do you have better ideas?

**Comment by sauloperez** [comment]
Option 2 sounds good to me.

**Comment by kirstenalarsen** [comment]
Option 3 - edit the text? Certainly the text under What do I Get seems excessive . .

**Comment by sauloperez** [comment]
and what about a "See more" link to show that "What do I get" section (maybe rephrased)?

**Comment by myriamboure** [comment]
I was just testifying while a user was setting up in front of me a hub, that when reaching the page "add promo image" she couldn't see the "next" button and the scroll didn't appear, so she just couldn't move on, she had to use tabulation to randomly "find" the next button to move to last page of the registration modal. I guess this is link to that issue, right? We are running on v1.18 so maybe some recent fix did solve that... but the scroll bar seems to randomly not appear on some computers... (I have also seen that on producer modals on the map, on some screens, scroll was not appearing).

**Comment by jazzdragon** [comment]
I'm going to take this issue, and implement the second solution above, I agree that it seems like better UX.

@myriamboure I'm not sure but I think what you're describing might deserve it's own ticket.

I don't have write access to this board, so if someone with permissions sees this, please assign me and move this ticket to "in dev". Thanks.

**Comment by sigmundpetersen** [comment]
Hey @jazzdragon great thanks! I moved it along to In Dev. To be assigned you need to be added to the GH organization by an admin 👍

**Comment by lin-d-hop** [comment]
@jazzdragon You should have an invite in your inbox. Once you accept you'll be able to assign yourself :-)
